app/lib/frontend/templates/views/pkg/package_list.mustache

```diff
@@ -16,7 +16,11 @@
         v <a href="{{& url}}">{{version}}</a>
         {{#show_dev_version}} / <a href="{{& dev_version_url}}">{{dev_version}}</a>{{/show_dev_version}}
         • updated: <span>{{last_uploaded}}</span>
-        {{#publisher_id}}• publisher: <a href="{{& publisher_url}}">{{publisher_id}}</a>{{/publisher_id}}
+        {{#publisher_id}}• <a href="{{& publisher_url}}"><img
```

Comment:
Also add a TODO to make this into a generic template.

Reply:
How so?